.storybook/webpack.config.js

```diff
@@ -3,6 +3,13 @@ const webpack = require('webpack');
 
 const staticPath = path.resolve(__dirname, '..', 'src', 'sentry', 'static', 'sentry');
 const componentPath = path.resolve(staticPath, 'app', 'components');
+const newSettingsPath = path.resolve(
```

Comment:
I would additionally create an alias to `path.resolve(staticPath, 'app')` so that we don't have to keep creating aliases and just make everything relative to `app`.